libraries/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -867,9 +882,35 @@ export class RushConfiguration {
       throw new InternalError('The default subspace was not created');
     }
 
+    // Look for rush projects in a seperate file first, and if it does not exist fallback onto rush.json
+
+    const resolvedRushProjectsFilename: string = `${this.rushJsonFolder}/${RushConstants.rushProjectsFilename}`;
+    let rushProjects: IRushConfigurationProjectJson[] | undefined;
+    try {
+      const rushProjectsJson: IRushProjectsJson = JsonFile.load(resolvedRushProjectsFilename);
```

Comment:
The JSON schema is not being validated here

Reply:
Added validation